spec/services/atmosphere/optimizer_spec.rb

```diff
@@ -115,8 +115,8 @@
                                   cpu: biggest_os_flavor.cpu,
                                   hdd: biggest_os_flavor.hdd,
                                   tenant: amazon)
-          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
-          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 100)
+          biggest_os_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
+          optimal_flavor.set_hourly_cost_for(Atmosphere::OSFamily.first, 1)
```

Comment:
Line 31 - `let(:amazon) { create(:amazon_with_flavors, funds: [fund]) }`.

IMHO default `lets` in specs are baaaaaaaaad but fixing this is out of scope of this PR. :)